.flake8

```diff
@@ -19,7 +19,10 @@ exclude = .git, tests/, build/,
           kaolin/cuda,
           kaolin/datasets,
           kaolin/engine,
-          kaolin/graphics,
+          kaolin/graphics/dib_renderer,
+          kaolin/graphics/DIBRenderer.py,
+          kaolin/graphics/NeuralMeshRenderer.py,
+          kaolin/graphics/nmr
```

Comment:
I'm not sure it have any effect (because it was already missing on `kaolin/helper.py`) but can you add a comma a the end of those two lines ?